### DISCUSSION TITLE
¿Como seria la modalidad para poder entregar el knock knock server?

### DISCUSSION THREAD

**Discussion by Nekuum** [hackatomik/SISCOx256#18] [General | Lab]
Tenia esa duda, ya que lo pude correr y abrir pero en general no sabría como entregar los demas laboratorios aparte de este, querria ver si podian contestarme esa duda

**Comment by JamSwiftDev** [comment]
lo que podrias hacer es crear diferentes carpetas donde almacenes los diferentes labotaroios ya que para correr un proyecto en especifico solamente tendriamos que cambiar de carpeta raiz y probar los diferentes proyectos y laboratorios te adjunto un ejemplo.
![Screenshot from 2024-09-05 13-01-44](https://github.com/user-attachments/assets/675cd7f1-373c-49cb-a5c5-6ac081ec76f1)
solamente tendrias que almacenar por el nombre de tus laboratorios y divirlo como en la imagen asi podrias subir todo en una misma rama para no complicarte tanto el asunto. Espero haberme explicado bien, si no, no dudes en decirme.
